src/utils/windowUtils.ts

```diff
@@ -15,13 +15,7 @@ export function warn(warning: string) {
     return;
```

Comment:
we should remove `Configs` and instead check `process.env.NODE_ENV !== "production"`